README.md

```diff
@@ -1,31 +1,44 @@
-# Fairing
+# Kubeflow Fairing
 
-Easily train ML models on Kubernetes, directly from your python code, or a Jupyter notebook.
+Easily package, deploy, and run your machine learning (ML) training jobs on Kubernetes,
+directly from Python code or a Jupyter notebook. 
+
+## Features
+
+Kubeflow Fairing provides functions to seemlessly train models remotely on Kubernetes
```

Comment:
I think the pitch should be "build, train, and deploy from anywhere"

Reply:
cc @abhi-g

Reply:
We can't quite claim "from anywhere." Packaging, deploying, and running jobs is how I interpret what Fairing does. I like "build", but I feel like that term collides with model development.  Here are a couple revisions, let me know what you think:
 
Easily package, deploy, and run your machine learning (ML) training jobs remotely,
directly from Python code or a Jupyter notebook. 

Easily build, deploy, and run your machine learning (ML) training jobs remotely,
directly from Python code or a Jupyter notebook. 

Easily build, deploy, and run your machine learning (ML) training jobs at scale on several environments,
directly from Python code or a Jupyter notebook. 

Easily package, deploy, and run your machine learning (ML) training jobs at scale,
directly from Python code or a Jupyter notebook.

Reply:
"Easily build, train, and deploy your machine learning (ML) models to Kubeflow
directly from Python code or a Jupyter notebook." ?

Reply:
@joeliedtke how about the one suggested by Karthik. The "Build/Train/Deploy" is how we have proposed the [CUJ] (http://bit.ly/cuj_train_deploy_notebook) which is on [Kubeflow Roadmap](https://github.com/kubeflow/kubeflow/blob/master/ROADMAP.md).  This has been reviewed by multiple users/customers over the course of the last quarter.